generators/app/index.js

```diff
@@ -346,7 +338,16 @@ module.exports = class extends BaseGenerator {
                 this.skipServer && (config.skipServer = true);
                 this.skipUserManagement && (config.skipUserManagement = true);
                 this.config.set(config);
+            },
+
+            insight() {
+                if (this.existingProject) return;
```

Comment:
If it is existing project send the data with a flag to indicate this is regeneration